canonical-web-and-design/ubuntu.com
# Add redirects to 4 Russian engage pages

## Done

Redirects 4 Russian engage pages to a URL following this format: `/engage/ru/english-title`
- the language should have been part of the URL from the start
- the Cyrillic characters in the URL make it look very odd when shared, i.e. https://ubuntu.com/engage/%D0%BE%D1%82%D1%87%D0%B5%D1%82-%D0%BE-%D1%86%D0%B5%D0%BD%D0%B0%D1%85-%D0%BD%D0%B0-%D0%B1%D0%B5%D1%81%D0%BF%D0%BB%D0%B0%D1%82%D0%BD%D1%8B%D0%B5-%D0%BE%D0%B1%D0%BB%D0%B0%D1%87%D0%BD%D1%8B%D0%B5-%D1%85%D1%80%D0%B0%D0%BD%D0%B8%D0%BB%D0%B8%D1%89%D0%B0

## QA

- Visit these pages and see that you are redirected to the correct engage page:

| Old URL | Redirects to |
| --- | --- |
| https://ubuntu-com-11254.demos.haus/engage/отчет-о-ценах-на-бесплатные-облачные-хранилища | https://ubuntu-com-11254.demos.haus/engage/ru/redhat-openstack-comparison-whitepaper |
| https://ubuntu-com-11254.demos.haus/engage/от-vmware-к-charmed-openstack | https://ubuntu-com-11254.demos.haus/engage/ru/vmware-to-charmed-openstack |
| https://ubuntu-com-11254.demos.haus/engage/сравнение-платформ-kubernetes | https://ubuntu-com-11254.demos.haus/engage/ru/enterprise-kubernetes-comparison |
| https://ubuntu-com-11254.demos.haus/engage/сравнение-платформ-red-hat-openstack-canonical-charmed-openstack | https://ubuntu-com-11254.demos.haus/engage/ru/cloud-pricing-report-2021 |
